prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.ts

```diff
@@ -61,10 +63,15 @@ export class EnrolleeOverviewComponent extends AdjudicationContainerComponent im
     this.hasActions = true;
   }
 
-  public onNavigateEnrollee(enrolleeId: number) {
+  public onNavigateEnrollee(enrolleeId: number): void {
     this.onRoute([enrolleeId, RouteUtils.currentRoutePath(this.router.url)]);
   }
 
+  public onRedirectCommunitySite(routePath: string | (string | number)[]): void {
+    const path = `${AdjudicationRoutes.MODULE_PATH}/${AdjudicationRoutes.SITE_REGISTRATIONS}/${routePath[0]}/${routePath[1]}/${routePath[2]}`;
```

Comment:
If you need to convert an array of strings to a URL you'll want to first check it's an array, and then use `join`.  You won't need the intermediate `const`.  This will handle an array when provided and make a string, and otherwise it's a string and you don't need to do anything.

```ts
if(Array.isArray(routePath)) {
  routePath = routePath.join('/');
}

this.router.navigate([routePath]);
```